share/man/man4/acpi_wmi.4

```diff
@@ -91,10 +97,14 @@ class Lenovo_BiosSetting : Lenovo_BiosElement {
       [WmiDataId(1), Description("BIOS setting")] String CurrentSetting;
       };
    ...
-
 .Ed
 .Sh SEE ALSO
 .Xr acpi 4
+.Sh STANDARDS
+.Rs
+.%T Windows Instrumentation: WMI and ACPI
+.%I Microsoft Corporation
+.Re
```

Comment:
The former link was dead and I can't find a link to the paper anymore, but I was able to find the name of the paper.

Reply:
There's a copy at https://github.com/microsoft/Windows-driver-samples/tree/main/wmi/wmiacpi